openpype/modules/deadline/repository/custom/plugins/GlobalJobPreLoad.py

```diff
@@ -190,17 +189,23 @@ def inject_openpype_environment(deadlinePlugin):
             raise RuntimeError(msg)
 
         if not os.environ.get("OPENPYPE_MONGO"):
-            print(">>> Missing OPENPYPE_MONGO env var, process won't work")
+            if not job.GetJobEnvironmentKeyValue('OPENPYPE_MONGO'):
```

Comment:
Why is `if not os.environ.get("OPENPYPE_MONGO"):` still checked when it's 100% time overriden by `job.GetJobEnvironmentKeyValue('OPENPYPE_MONGO')`?